### PR TITLE
remove runCom1DFA function inside and call com1DFAMain

### PR DESCRIPTION
**Pull request by awirb**
* update runScripts
* add pytest

### CODE REVIEW

File: avaframe/com1DFA/com1DFA.py [hunk 1]
```diff
@@ -37,7 +43,106 @@
...
+    modCfg, variationDict = dP.getParameterVariationInfo(avalancheDir, com1DFA, cfgFile, variationDict)
+
+    # fetch input data - dem, release-, entrainment- and resistance areas
+    inputSimFiles = gI.getInputDataCom1DFA(avalancheDir, modCfg['FLAGS'])
+
+    # write full configuration file to file
+    cfgUtils.writeCfgFile(avalancheDir, com1DFA, modCfg, fileName='sourceConfiguration')
+
+    # create a list of simulations and generate an individual configuration object for each simulation
+    # if need to reproduce exactely the hash - need to be strings with exactely the same number of digits!!
```
Comment (fso42) [inline avaframe/com1DFA/com1DFA.py:88]: Typo: exactly

File: avaframe/in3Utils/cfgUtils.py [hunk 2]
```diff
@@ -102,10 +102,14 @@ def getModuleConfig(module, fileOverride='', modInfo=False):
...
-
-    return cfg
+    if modInfo:
+        # Finally read it
+        cfg, modDict = compareConfig(iniFile, modName, compare, modInfo)
+        return cfg, modDict
+    else:
+        # Finally read it
+        cfg = compareConfig(iniFile, modName, compare, modInfo)
+        return cfg
```
Comment (fso42) [inline avaframe/in3Utils/cfgUtils.py:112]: I suggest: 
```suggestion
    # Finally read it
    cfg, modDict = compareConfig(iniFile, modName, compare, modInfo)
    if modInfo:
        return cfg, modDict
    return cfg
```

File: avaframe/runScripts/runTestFP.py [hunk 3]
```diff
@@ -10,7 +10,6 @@
 import avaframe.in3Utils.fileHandlerUtils as fU
 from avaframe.in1Data import getInput as gI
 import avaframe.com1DFA.com1DFA as com1DFA
-from avaframe import runCom1DFA
 import avaframe.ana1Tests.FPtest as FPtest
```
Comment (fso42) [inline avaframe/runScripts/runTestFP.py:13]: This script fails with:
`ModuleNotFoundError: No module named 'avaframe.out3Plot.makePalette'`

and the com1DFA import is also not the correct one, right?

**Review by fso42 — CHANGES_REQUESTED** [review]

File: avaframe/com1DFA/com1DFA.py (hunk 1 above)
Comment (awirb) [reply avaframe/com1DFA/com1DFA.py:88]: done

File: avaframe/in3Utils/cfgUtils.py (hunk 2 above)
Comment (awirb) [reply avaframe/in3Utils/cfgUtils.py:112]: this is not possible as compareConfig only returns the modDict if modInfo is true

File: avaframe/runScripts/runTestFP.py (hunk 3 above)
Comment (awirb) [reply avaframe/runScripts/runTestFP.py]: I remove the import as it is not required and changed the import of com1DFA, also removed the hSPH as it is not added to the particles anymore - but still fails -made an issue out of it

File: avaframe/in3Utils/cfgUtils.py (hunk 2 above)
Comment (matthiasto) [reply avaframe/in3Utils/cfgUtils.py:112]: you could just always return the modDict. This way felixs solution works and is shorter and nicer and if you call this function somewhere else and don't want to use the modDict just do:
cfg, _ = compareConfig(iniFile, modName, compare, modInfo)

**Review by matthiasto — CHANGES_REQUESTED** [review]

File: avaframe/in3Utils/cfgUtils.py (hunk 2 above)
Comment (awirb) [reply avaframe/in3Utils/cfgUtils.py]: changed now modDict is always returned and updated the calls where it is not required

**Review by matthiasto — APPROVED** [review]

**Review by fso42 — APPROVED** [review]

**Pull request merged**